knora/dsplib/models/listnode.py

```diff
@@ -184,7 +184,7 @@ def __init__(self,
         self._project = project.id if isinstance(project, Project) else str(project) if project is not None else None
         self._id = str(id) if id is not None else None
         self._label = LangString(label)
-        self._comments = LangString(comments)
+        self._comments = LangString(comments) if comments is not None else None
```

Comment:
any reason you don't just write `LangString(comments) if comments else None`?

Reply:
also, technically this violates typing here: `self._comments` should be `Optional[LangString]`, not just `LangString`, if it can be `None`

Reply:
and on that note `Union[str, None]` should probably be replaced with `Optional[str]`